infrastructure/k8s/helm-chart/eoapi/values.yaml

```diff
@@ -1,18 +1,88 @@
 # default values for eoapi
 
-gitsha_comment: >
-  To pass `gitSha` in during `helm template` or `helm install` do:
+comment: >
+  some postgres variables are basically secrets that should be passed in via os environment variables.
+  For example, to pass username or password information during `helm template` or `helm install` do:
 
-  $ helm install
-      --set gitSha=$(git rev-parse HEAD | cut -c1-10)
+  $ export GITSHA=$(git rev-parse HEAD | cut -c1-10)
+  $ export PGUSER_VAR=s00pers3cr3t
+  $ export POSTGRES_USER_VAR=s00pers3cr3t
+  $ export POSTGRES_PASSWORD_VAR=superuserfoobar
+  $ export PGPASSWORD_VAR=foobar
+
+  $ helm install \
+      --namespace eoapi \
+      --create-namespace \
+      --set gitSha=$GITSHA \
+      --set db.settings.secrets.PGUSER=$PGUSER \
+      --set db.settings.secrets.POSTGRES_USER=$POSTGRES_USER \
+      --set db.settings.secrets.POSTGRES_PASS=$POSTGRES_PASS \
+      --set db.settings.secrets.PGPASSWORD=$PGPASSWORD \
+      --set db.settings.secrets.POSTGRES_PASSWORD=$POSTGRES_PASSWORD \
+      --set db.settings.secrets.POSTGRES_PASS=$PGPASSWORD \
```

Comment:
@ranchodeluxe what's going on here? why do we have 4 different ways we're setting the postgres password?

Reply:
Good Question!

My understanding is that one is for the superuser and other for app user

Reply:
welp, there was type so now only three 😆

Reply:
@batpad : handled this with some acrobatics